calico-enterprise_versioned_docs/version-3.16/releases.json

```diff
@@ -1,4 +1,257 @@
 [
+  {
+    "title": "v3.16.3",
+    "tigera-operator": {
+      "image": "tigera/operator",
+      "version": "v1.29.4",
```

Comment:
Will likely be 1.29.5, for @danudey to confirm.

Reply:
Fixed.